Tests/Functional/FailureMessagesTestCase/main.swift

```diff
@@ -121,6 +123,20 @@ class FailureMessagesTestCase: XCTestCase {
         XCTAssertGreaterThanOrEqual(-1, 0, "message", file: "test.swift")
     }
 
+// CHECK: Test Case 'FailureMessagesTestCase.testAssertIdentical' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+// CHECK: test.swift:[[@LINE+3]]: error: FailureMessagesTestCase.testAssertIdentical : XCTAssertIdentical failed: \("a"\) is not identical to \("b"\) - message
+// CHECK: Test Case 'FailureMessagesTestCase.testAssertIdentical' failed \(\d+\.\d+ seconds\)
+    func testAssertIdentical() {
+        XCTAssertIdentical("a" as AnyObject, "b" as AnyObject, "message", file: "test.swift")
+    }
+
+// CHECK: Test Case 'FailureMessagesTestCase.testAssertNotIdentical' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+// CHECK: test.swift:[[@LINE+3]]: error: FailureMessagesTestCase.testAssertNotIdentical : XCTAssertNotIdentical failed: \("nil"\) is identical to \("nil"\) - message
+// CHECK: Test Case 'FailureMessagesTestCase.testAssertNotIdentical' failed \(\d+\.\d+ seconds\)
+    func testAssertNotIdentical() {
+        XCTAssertNotIdentical(nil, nil, "message", file: "test.swift")
```

Comment:
Thank you for this change! I think we can land it soon

My only request is that you add a few more assertions to these two tests, in particular **success** cases which validate that both `XCTAssertIdentical` and `XCTAssertNotIdentical` _succeed_ when passed identical or not identical values, respectively. Currently, the tests only validate failure scenarios, but we should be sure to have tests covering when they pass as well. Personally, I'm fine just adding those inside the existing two test method bodies, to keep things simple.

Reply:
Oh that's a good point. I added a few succeeding assertions to the test.